scripts/generate-social-banner-rangelink.py

```
@@ -0,0 +1,118 @@
+    new_w = int(icon.width * scale)
+    new_h = int(icon.height * scale)
+    return icon.resize((new_w, new_h), Image.LANCZOS)
+
+
+def main():
+    meta = load_project_meta()
+    icon_url = resolve_icon_url(meta)
+    icon = fetch_icon(icon_url)
+    icon = sized_icon(icon, cfg.ICON_SIZE)
+
+    bannertitle = meta.get("bannertitle") or meta["title"].rstrip(" Extension")
```

Ruff (0.15.15)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)